messaging-activemq/src/main/java/org/wildfly/extension/messaging/activemq/ServerDefinition.java

```diff
@@ -699,7 +699,7 @@
      * @see ActiveMQDefaultConfiguration#getDefaultIdCacheSize
      */
     public static final SimpleAttributeDefinition ID_CACHE_SIZE = create("id-cache-size", INT)
-            .setDefaultValue(new ModelNode(20000))
+            .setDefaultValue(new ModelNode(ActiveMQDefaultConfiguration.getDefaultIdCacheSize()))
```

Comment:
Here you are introducing a new problem by converting a static value to a changeable one.